### ISSUE TITLE
Include photos from `xrpics` repo

### ISSUE THREAD

**Issue by cvan** [WebVRRocks/webvrrocks#255]
https://github.com/trevorfsmith/xrpics

@trevorfsmith: awesome, awesome work on this! 👍 is it cool if I include some of the photos on https://webvr.rocks/ for all the WebVR-compatible headsets (and gamepad mappings)?

**Comment by TrevorFSmith** [comment]
Aw, thanks!

Yep, there're open licensed for any use, with or without retribution... I mean attribution. 😁

**Comment by cvan** [comment]
yep, saw in your README. just wanted to make sure before I published it on WebVR Rocks, since quite a few people refer to it. thanks for the generosity! 😄

**Comment by TrevorFSmith** [comment]
I was doing it for Spaciblō anyway, so it seemed silly to not share.
